crates/kicad-parser/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "kicad-parser"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+sexp = "1.1.4"
+opencascade = { version = "0.1", path = "../opencascade" }
+glam = { version = "0.23", features = ["bytemuck"] }
```

Comment:
I think for the `kicad-parser` crate specifically, it can just be focused on parsing the data into nice Rust structs.

We can have another module in the `opencascade` crate specifically for generating shapes from the data this crate pulls in. So even `glam` is probably unnecessary as a dependency here, since we're only using it to construct `dvec3` from the parsed data without doing any math.

This crate can keep things in their original parsed 2D coordinates, and it can be the responsibility of the `opencascade` layer to place those 2D "sketches" in the desired workplane (Z might not be `0` for all use cases!).

I'd be happy to make this change, or stub out what I think the higher level kicad code would look like.

Reply:
Ah okay makes sense. So remove all the `Into` primitives too? I guess moving those into another '2d graphics' crate means they can be used by another 2d import such as .dxf.

I'll hand it to you since it sounds like you have an idea of what you're expecting.

Reply:
> So remove all the Into primitives too?

Likely so, I think that will happen at a higher level. Sorry for being somewhat vague but I'll try to follow up on this to show what I was thinking for the design.